src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,2368 @@
+			}
+			else
+			{
+				Sat_press_val = exp(C8 / TK + C9 + C10 * TK + C11 * pow(TK, 2) + C12 * pow(TK, 3) + C13 * log(TK)) / 1000;
+			}
+			return Sat_press_val;
+
+		}*/
+		/*		Real64 Model::CalcHum_ratio_W(Real64 Tdb, Real64 RH, Real64 P)
+		{
+			// SUBROUTINE INFORMATION:
+			//       AUTHOR         Spencer Maxwell Dutton
+			//       DATE WRITTEN   October 2017 
+			//       MODIFIED       
+			//       RE-ENGINEERED  na
+
+			// PURPOSE OF THIS SUBROUTINE:
+			// Function to calculate humidity ratio [kg H2O/kg air]
+			// Given dry bulb and wet bulb temperature inputs [degC]
+
+			// METHODOLOGY EMPLOYED:
+			// ASHRAE Fundamentals handbood (2005)
+			// Tdb = Dry bulb temperature [degC]
+			// RH = Relative Humidity [Fraction or %]
+			// P = Ambient Pressure [kPa] 
+
+			// REFERENCES:
+			// na
+
+			// Using/Aliasing
+
+			Real64 Pws = Sat_press(Tdb);
+			Real64 Hum_rat = 0.62198 * RH * Pws / (P - RH * Pws);   // Equation 22, 24, p6.8
+			return Hum_rat;
+		}*/
+
+
+
+		Real64 Model::CheckVal_W(Real64 W, Real64 T, Real64 P)
+		{
+			//P must be in pascals NOT kPa
+			Real64 OutletRHtest = PsyRhFnTdbWPb(T, W, P); //could also use outlet pressure instead of fixed
+			Real64 OutletW = PsyWFnTdbRhPb(T, OutletRHtest, P, "Humidity ratio exceeded realistic range error called in "+ Name + ", check performance curve");
+			return OutletW;
+		}
+		Real64 Model::CheckVal_T(Real64 T)
+		{
+			if ((T > 100) || (T < 0))
+			{
+				ShowWarningError("Supply air temperature exceeded realistic range error called in " + Name + ", check performance curve");
+			}
+			return T;
+		}
+		bool Model::SetStandByMode(CMode Mode0, Real64 Tosa, Real64 Wosa, Real64 Tra, Real64 Wra)
+		{
+			// SUBROUTINE INFORMATION:
+			//       AUTHOR         Spencer Maxwell Dutton
+			//       DATE WRITTEN   October 2017 
+			//       MODIFIED       
+			//       RE-ENGINEERED  na
+
+			// PURPOSE OF THIS SUBROUTINE:
+			// Set the supply air mass flow rate, power use, and all the other parameters for a setting. 
+
+			// METHODOLOGY EMPLOYED:
+			// Uses the relevant lookup take to specify the parameters, or uses default conditions.
+			// In setting the supply air temperature for now just use return air future improvement will use look up table
+
+
+			// REFERENCES:
+			// na
+
+			// Using/Aliasing
+			// using the pointer to the first of the operating modes (which is always the standby mode by definition.
+			//Get a handel to the map of possible solutions (valid combinations of OSA fraction and Msa)
+			CModeSolutionSpace solutionspace = Mode0.sol;
+			//assess the sixe of the solution space to make sure it holds at least 1 combination
+			int solution_map_sizeX = solutionspace.PointX.size();
+			//Check the normalization reference is set if not set a default
+			Real64 NormalizationReference = GetNormalPoint(1);// assumes the model has at least 1 curve and that all are set the same, this is explained in the IO reference

```

Comment:
I need some clarification on this.  What do you mean it assumes the model has at least one curve?  The IDF could be littered with curves!  And you are calling it to get a normalization reference for the first one??  I don't understand this.